### DISCUSSION TITLE
What is the coordinate system used in CARLA?

### DISCUSSION THREAD

**Discussion by joe-source113898** [carla-simulator/carla#6842] [Q&A]
I want to convert the coordinate system of CARLA in a right-handed coordinate system. I'm beginner in this simulator. Thanks in advance.
I tried this approach, but it didn´t work for me: https://github.com/carla-simulator/ros-bridge/issues/637

**Comment by Spyridonidis98** [comment]
carla is build with unreal engine so it uses unreal engines coordinate system which is the left handed coordinate system with the x axis looking forward y axis looking right and z axis looking up

**Reply by nveshaan** [reply]
Even though it is a left-handed coordinate system, the position, velocity, etc. of a vehicle you get are in right-handed system

**Reply by Spyridonidis98** [reply]
That is not the case, i have personally collected vehicles data from the simulator and the actors position is always in the coordinate system that carla uses which is the left handed.

**Reply by nveshaan** [reply]
Oh is it?

I used carla 0.10.0 UE5, maybe there was a change.

Or I might have unknowingly converted it beforehand.
